modules/electromagnetics/test/tests/kernels/vector_helmholtz/vector_ADmaterial_wave_equation_convergence.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Script to run spatial convergence study on vector_ADmaterial_wave_equation.i
+
+import mms
+import sympy
+import os
+
+if os.path.isfile('../../../../electromagnetics-opt'):
+    executable = '../../../../electromagnetics-opt'
+else:
+     executable = '../../../../../combined/combined-opt'
+
+df1 = mms.run_spatial('vector_ADmaterial_wave_equation.i', 6, console=False, executable=executable, x_pp='h', y_pp=['error_real', 'error_imag'],mpi=1)
+
+fig = mms.ConvergencePlot(xlabel='Element Size ($h$)', ylabel='$L_2$ Error')
```

Comment:
See Navier Stokes MMS testing. You want to add a check on the order of convergence, and error if it changes

Reply:
@GiudGiud I can add a check on the order of convergences, but I have a clarification question first:
- The current test for the `*_convergence.py` files is a `CSVDiff`. Wouldn't checking the difference in CSV file include any changes in the order of convergences?

Reply:
ok yes this is fine. 
I m more used to just checking the order, but the csv of the output includes that